src/bb-modules/Theme/Service.php

```diff
@@ -276,7 +276,8 @@ public function getThemes($client = true)
         $path = $this->getThemesPath();
         if ($handle = opendir($path)) {
             while (false !== ($file = readdir($handle))) {
-                if (is_dir($path . DIRECTORY_SEPARATOR . $file) && $file{0} != '.') {
+//**Deprecated: Array and string offset access syntax with curly braces is deprecated in /var/www/vhosts/webbhostingservices.com/httpdocs/boxbilling/src/bb-modules/Theme/Service.php on line 279 *//
```

Comment:
Remove references to your development environment.

/var/www/vhosts/webbhostingservices.com/httpdocs/boxbilling/src/bb-modules/Theme/Service.php on line 279 */